### PR TITLE
Maintenance/update docs

### PR DESCRIPTION
**Pull request by Ilyesbdlala**
Updated documentation in the following locations:
- ParseDefinition: updated the example
- parser (directory): updated the Docker image alpine version 12 -> 14.
-  ScanType: updated the example

### CODE REVIEW

File: docs/contributing/integrating-a-scanner/parser-dir.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ For our JavaScript Parser SDK the Dockerfile should look like this:
```
Comment (SebieF) [inline docs/contributing/integrating-a-scanner/parser-dir.md:15]: Would be cool to add a note that if your parser does not depend on any additional packages (no package.json file required), a reduced Dockerfile for the parser can be used:
```
ARG namespace

ARG baseImageTag

FROM ${namespace:-securecodebox}/parser-sdk-nodejs:${baseImageTag:-latest}  

WORKDIR /home/app/parser-wrapper/parser/  

COPY --chown=app:app ./parser.js ./parser.js
```

**Review by SebieF — COMMENTED** [review]
Good changes, one addition could be made :)

File: docs/contributing/integrating-a-scanner/parser-dir.md (hunk 1 above)
Comment (Ilyesbdlala) [reply docs/contributing/integrating-a-scanner/parser-dir.md:15]: Yeah that's a good suggession. I added it to the PR.

**Review by SebieF — APPROVED** [review]
:+1:

File: docs/api/crds/parse-definition.md [hunk 2]
```diff
@@ -37,8 +37,8 @@ kind: ParseDefinition
...
   name: zap-json
 spec:
-  image: docker.io/securecodebox/parser-zap
-  imagePullSecrets:
-  - name: dockerhub-token
-  ttlSecondsAfterFinished: 60
+  image: "{{ .Values.parser.image.repository }}:{{ .Values.parser.image.tag | default .Chart.Version }}"
+  ttlSecondsAfterFinished: {{ .Values.parser.ttlSecondsAfterFinished }}
+  env: 
+    {{- toYaml .Values.parser.env | nindent 4 }}
```
Comment (J12934) [inline docs/api/crds/parse-definition.md:43]: This example is great for users wanting to integrate new scanners into our repo, but is incorrect in my opinion at this page.
This API page describes the ParseDefinition CRD and which values are supported by it.

The helm templates you changed it to, describe the helm templates we use in our helm charts by convention, but its not required for users to use the same conventions. I'd prever to keep this page focussed on the actual crd and add a note like: "If you are adding new ParseDefinitions to the secureCodeBox repo, see the conventions used for the helm charts of integrated scanner: <link to contributions page." 

This example should then be moved to the contributions page (if it doesn't exist there already)

**Review by J12934 — CHANGES_REQUESTED** [review]

File: docs/api/crds/parse-definition.md (hunk 2 above)
Comment (Ilyesbdlala) [reply docs/api/crds/parse-definition.md]: Yeah I agree, I just changed to that way now,

**Review by fuhrmeistery — APPROVED** [review]

**Pull request merged**